src/test/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/SemanticAnalyzerOperatorTest.java

```diff
@@ -55,9 +55,9 @@ public void compareSubstringFunctionCallEqualsToNumberShouldFail() {
     }
 
     @Test
-    public void compareLogAndAbsFunctionCallWithIntegerSmallerThanStringShouldFail() {
+    public void compareLogFunctionCallWithIntegerSmallerThanStringShouldFail() {
```

Comment:
Could you add the exception expectation?

Reply:
Ok will do.